### ISSUE TITLE
Importing data into FHIR store failing 

### ISSUE THREAD

**Issue by sanjaykhobragade** [GoogleCloudPlatform/healthcare#124]
curl -X POST     -H "Authorization: Bearer $(gcloud auth print-access-token)"     -d '{"gcsSourceLocation":{"gcsUri":"gs://'${PROJECT_ID?}'/'demo_data.ndjson'"}}'     "https://healthcare.googleapis.com/${API_VERSION?}/projects/${PROJECT_ID?}/locations/${REGION?}/datasets/${DATASET_ID?}/fhirStores/${FHIR_STORE_ID?}:import"
{
  "error": {
    "code": 400,
    "message": "Invalid JSON payload received. Unknown name \"{\"gcsSourceLocation\":{\"gcsUri\":\"gs://dicom-poc/demo_data.ndjson\"}}\": Cannot bind query parameter. Field '{\"gcsSourceLocation\":{\"gcsUri\":\"gs://dicom-poc/demo_data' could not be found in request message.",
    "status": "INVALID_ARGUMENT",
    "details": [
      {
        "@type": "type.googleapis.com/google.rpc.BadRequest",
        "fieldViolations": [
          {
            "description": "Invalid JSON payload received. Unknown name \"{\"gcsSourceLocation\":{\"gcsUri\":\"gs://dicom-poc/demo_data.ndjson\"}}\": Cannot bind query parameter. Field '{\"gcsSourceLocation\":{\"gcsUri\":\"gs://dicom-poc/demo_data' could not be found in request message."
          }
        ]
      }
    ]
  }
}

Not sure why it's looking for demo_data instead of demo_data.ndjson resource

**Comment by nikklassen** [comment]
It looks like the issue is the API isn't interpreting your request as JSON. You'll need to add `-H "Content-Type: application/json"`. I'll update the documentation to reflect this. Also, if you're using the v1beta1 version of the API the request format has changed. [docs](https://cloud.google.com/healthcare/docs/reference/rest/v1beta1/projects.locations.datasets.fhirStores/import). You'll need to change the request to
```
curl -X POST -H "Authorization: Bearer $(gcloud auth print-access-token)" -d '{"gcsSource":{"uri":"gs://'${PROJECT_ID?}'/'demo_data.ndjson'"}}' "https://healthcare.googleapis.com/${API_VERSION?}/projects/${PROJECT_ID?}/locations/${REGION?}/datasets/${DATASET_ID?}/fhirStores/${FHIR_STORE_ID?}:import"
```

I'll also fix this in the doc.

**Comment by nikklassen** [comment]
Closed in #135
